File: backend_time_management/app/Models/EventModel.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class EventModel extends Model
{
    protected $fillable = [
        'event',
        'date',
        'time',
    ];
    use HasFactory;
}

File: time_management_application/lib/models/event_model.dart
class EventModel {
  String? event;
  String? date;
  String? time;

  EventModel(this.event, this.date, this.time);
  EventModel.fromJson(Map<String, dynamic> json) {
    event = json["event"];
    date = json["date"];
    time = json["time"];
  }
}
